nltk/classify/rte_classify.py

```diff
@@ -70,8 +70,8 @@ def __init__(self, rtepair, stop=True, lemmatize=False):
         self.hyp_words = set(self.hyp_tokens)
 
         if lemmatize:
-            self.text_words = set(lemmatize(token) for token in self.text_tokens)
```

Comment:
It looks like the wrong fix, but it's clear that this has never been used or tested!

Reply:
@jhelie: sorry for the slow response. I like @jnothman's suggestion to rename the parameter. The name `use_lemmatize` looks good to me. Thanks.